app/controllers/api/members_controller.rb

```diff
@@ -16,4 +22,13 @@ def create
   def member_params
     params.permit(:name, :email, :country, :postal)
   end
+
+  def member_validation
+    [
+      { name: 'email', data_type: 'email', required: true },
+      { name: 'country', data_type: 'country', required: false },
+      { name: 'postal', data_type: 'postal', required: false },
+      { name: 'name', data_type: 'text', required: true }
+    ]
+  end
```

Comment:
I believe this controller has too much logic into it that doesn't belong in there: what fields to validate and how, executing `send_to_ak` if successful. I think moving all that logic into a service might be a good idea.